lib/utils/handleKeyCommand.js

```diff
@@ -0,0 +1,11 @@
+export const handleKeyCommand = (handler, disabled) => {
```

Comment:
I'd suggest "main arg, options" pattern instead
```suggestion
export const handleKeyCommand = (handler, { disabled }) => {
```